File: solutions/typescript/VB_T2_013_get_char_code.ts
// VB-T2-013: Get Char Code -- TypeScript baseline
function getCharCode(s: string, index: number): number { return s.charCodeAt(index); }
console.assert(getCharCode("A", 0) === 65);
console.assert(getCharCode("hello", 0) === 104);
console.assert(getCharCode("hello", 4) === 111);
console.assert(getCharCode("Z", 0) === 90);

File: solutions/python/VB_T2_013_get_char_code.py
"""VB-T2-013: Get Char Code -- Python baseline."""
def get_char_code(s: str, index: int) -> int:
    return ord(s[index])

if __name__ == "__main__":
    assert get_char_code("A", 0) == 65
    assert get_char_code("hello", 0) == 104
    assert get_char_code("hello", 4) == 111
    assert get_char_code("Z", 0) == 90
    print("VB-T2-013 ok")
